lib/realtime/vehicle.ex

```diff
@@ -69,6 +71,8 @@ defmodule Realtime.Vehicle do
     :operator_id,
     :operator_name,
     :run_id,
+    :sources,
+    :data_discrepancies,
```

Comment:
`[]` would be a reasonable and valid default for this field, so I don't think we need to require the field.